site/content/docs/main/release-instructions.md

```diff
@@ -1,16 +1,103 @@
 ---
 title: "Release Instructions"
 layout: docs
+toc: "true"
 ---
-
 This page covers the steps to perform when releasing a new version of Velero.
 
-## Preparing for a release
-
-### (GA Only) Create release blog post
-
-For each major or minor release, create and publish a blog post to let folks know what's new.
-
+## General notes
+- Please read the documented variables in each scrip to understand what they are for and how to properly format their values.
+- You will need to have a remote named `origin` pointing to the Velero repository so the scripts can work properly.
+- GA release: major and minor releases only. Example: 1.0 (major), 1.5 (minor).
+- Pre-releases: Any release leading up to a GA. Example: 1.4.0-beta.1, 1.5.0-rc.1
+- RC releases: Release Candidate, contains everything that is supposed to ship with the GA release. This is still a pre-release.
+
+## Preparing
+
+### Create release blog post (GA only)
+For each major or minor release, create and publish a blog post to let folks know what's new. Please follow these [instructions](Instructions-to-write-and-release-a-blog-post).
+
+### Changelog and Docs PR
+#### Troubleshooting
+- `You don't have enough free space in /var/cache/apt/archives/` when running `make serve-docs`: run `docker system prune`.
```

Comment:
I think it would be clearer if this line had something before the error message like:
```suggestion
- If you encounter the error `You don't have enough free space in /var/cache/apt/archives/` when running `make serve-docs`: run `docker system prune`.
```